### PR TITLE
adding my styleguides here since it is linked with pontoon

### PR DESCRIPTION
**Pull request by talnetd**
We have finished a short and insightful stylegudie for locale my 🇲🇲

### CODE REVIEW

**Comment by talnetd** [comment]
@laminko  adding laminko from 🇲🇲 community to review. Please La Min Ko review and check if this is ok as part of the process from community guidelines.

Thanks
TAL

**Comment by talnetd** [comment]
Fixing the check fail from travis.

**Comment by laminko** [comment]
Reviewed and ok.

**Comment by talnetd** [comment]
@MikkCZ  anything else we can help make this into mainline ?

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,4 @@
 _book
 l10ndocs-linter
 node_modules
+package-lock.json
```
Comment (MikkCZ) [inline .gitignore]: `package-lock.json` should be committed to the repository. Please revert this line.

File: package-lock.json [hunk 2]
```diff
@@ -1,4698 +0,0 @@
-{
```
Comment (MikkCZ) [inline package-lock.json]: Please do not remove this file.

**Comment by talnetd** [comment]
Are we good yet ?

**Comment by talnetd** [comment]
Will work on this change request. @laminko please help me address the topics and if there is none in our locale, put back the English one.

**Comment by laminko** [comment]
sure ko @talnetd , i will put back original one.

**Pull request merged**